bindings/c/test/fdb_c_client_config_tests.py

```diff
@@ -291,7 +279,13 @@ def run_tests(self):
         help="Path to the fdb_c_client_config_tester executable.",
         required=True,
     )
+    parser.add_argument("unittest_args", nargs=argparse.REMAINDER)
+
     args = parser.parse_args()
-    test = ClientConfigTests(args)
-    failed_cnt = test.run_tests()
-    sys.exit(failed_cnt)
+    sys.argv[1:] = args.unittest_args
+
+    downloader = FdbBinaryDownloader(args.build_dir)
+    downloader.download_old_binaries(PREV_RELEASE_VERSION)
+    downloader.download_old_binaries(PREV_PREV_RELEASE_VERSION)
```

Comment:
There were some platform (x86) checks before. Are those no longer necessary?

Reply:
The test suite is defined to be executed only for x86 in cmake lists. So this check is redundant.